modules/nlu/src/backend/tools/stopWords.ts

```diff
@@ -0,0 +1,39 @@
+import { createReadStream } from 'fs'
+import fs from 'fs-extra'
+import path from 'path'
+import readline from 'readline'
+
+fs.exists
```

Comment:
Leftover. Also, I would suggest "fse" instead of "fs" so we can easily differentiate between the two.